styles.css

```diff
@@ -0,0 +1,98 @@
+/* Topnav CSS */
+.topnav {
+  display: flex;
+  width: inherit;
+  height: 72px;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  font-family: "Inter", sans-serif;
+  float: left;
+  color: #172b4d;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.topnav a:hover {
+  color: #36b37e;
+}
+
+/* Add a color to the active/current link */
+.topnav a.active {
+  color: #36b37e;
+  text-decoration: underline;
+}
+
+/* FLexbox for the header */
+.right {
+  flex-grow: 1;
+}
+
+.left {
+  flex-grow: 3;
+}
+
+.nav-menu {
+  flex-grow: 2;
+}
+
+/* Headline CSS */
+.background-image {
+  display: flex;
+}
+
+.headline {
+  background-image: url('ressources/5243835.jpg');
```

Comment:
- [ ] **[OPTIONAL]** While it's not mentioned that you are obligated to use the background image provided by the Figma Template, I highly recommend you to do so, as the template was planned by a great developer, to make the entire page to fit together as one. This will enhance the trustfulness of your page!